pkg/network/cluster_config.go

```diff
@@ -14,6 +14,8 @@ import (
 func ValidateClusterConfig(clusterConfig configv1.NetworkSpec) error {
 	// Check all networks for overlaps
 	pool := iputil.IPPool{}
+	// Maintain a list of plugins that use hostPrefix
+	pluginsUsingHostPrefix := []string{"OpenShiftSDN", "OVNKubernetes"}
```

Comment:
These are defined as constants, `operv1.NetworkTypeOpenShiftSDN` and `operv1.NetworkTypeOVNKubnernetes`

Reply:
ack.